iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/RetrieveRemoteResourcesCheck.java

```diff
@@ -44,17 +44,23 @@ public class RetrieveRemoteResourcesCheck implements IacCheck {
   private static final String CURL = "curl";
 
   private static final List<String> WGET_AUTH_FLAGS = List.of("--http-user", "--http-password", "--proxy-user", "--proxy-password", "--load-cookies");
+  private static final List<String> WGET_REQUEST_FLAGS = List.of("--header", "--method", "--body-data", "--referer", "--save-headers",
```

Comment:
IMO it can be in one list named `WGET_FORBIDDEN_FLAGS` or similar

Reply:
I was hesitating between the 2, I'll change it thanks!